sakaar/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('heroes.urls')),
```

Comment:
I suggest you to make separate api_urls.
e.g
```suggestion
    path('api/', include('saakar.api_urls')),
```